testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -25,6 +32,7 @@
 When(/^I click on "([^"]+)" in row "([^"]+)"$/) do |link, item|
   within(:xpath, "//tr[td[contains(.,'#{item}')]]") do
     click_link_or_button(link)
+    wait_for_spa_transition(link)
```

Comment:
hmmmm Is a bit dangerous the way we handle the transition time here :/
As we have a lot more lines of code in the test suite calling `.click` and in theory, we should perform that wait on each of them.
We need to think of a way to trigger it without being forced to call that method every time. TBH not sure yet how we can handle it properly.

Reply:
Taking into account that we have a timeout on every Caybara finder and actions like click, it might be better to keep that transition transparent from the point of view of the testsuite.
Just as we do now, when a click triggers a load of a new page.

Reply:
Well, I added it exactly because we had 200 failures :P 

We have now no full refresh between links, that's probably the reason for Caybara to wait less time between clicks, even having timeouts in most of the finders. I will do some more investigation on this.

But yeah, I agree, we can try to find a better way of adding this method.